model/host/db.go

```diff
@@ -21,7 +21,8 @@ import (
 
 const (
 	// Collection is the name of the MongoDB collection that stores hosts.
-	Collection = "hosts"
+	Collection        = "hosts"
+	VolumesCollection = "volumes"
```

Comment:
Do we need a new collection for these? Would it work to embed them in the host doc?

Reply:
This was more a conversation when Thomas first started this, but yes we need a new collection because volumes are created independent of hosts and then later detached, so they can't be tied to the host collection.